app/src/features/FhirAttributePanel/AttributeInputGroup.tsx

```diff
@@ -211,34 +189,37 @@ const AttributeInputGroup = ({
           >
             <Grid item>
               <Button
-                {...bindTrigger(popupState)}
                 size="small"
                 variant={"outlined"}
                 className={classes.button}
-                startIcon={<Add />}
-                onClick={handleMenuClick}
+                startIcon={
+                  <Icon
+                    icon={IconNames.ALIGN_LEFT}
+                    className={classes.blueprintIcon}
```

Comment:
It doesn't seem like a bad idea, but maybe best would be to set dynamically this color, because if we want to set the start icon in a different color it won't be automatized...